operate/schema/src/main/java/io/camunda/operate/schema/SchemaStartup.java

```diff
@@ -71,6 +71,8 @@ public void initializeSchema() throws MigrationException {
         LOGGER.info(
             "SchemaStartup: schema won't be created, it either already exist, or schema creation is disabled in configuration.");
       }
+      schemaManager.checkAndUpdateIndices();
```

Comment:
Should we may be check the property [`createSchema`](https://github.com/camunda/zeebe/blob/a24d54f360ea52a84d92884d416ff433d04e487e/operate/schema/src/main/java/io/camunda/operate/schema/SchemaStartup.java#L62), and only update when it's true? I don't have strong opinion here and don't expect many problems if we don't, but this is currently the way to select one node in cluster setup that is responsible for schema changes.